common/models/config-file.js

```diff
@@ -305,17 +303,16 @@ module.exports = function(ConfigFile) {
     assert(Array.isArray(configFiles));
     var configFile;
     var results = [];
-    for(var i = 0; i < configFiles.length; i++) {
+    for (var i = 0; i < configFiles.length; i++) {
       configFile = configFiles[i];
       // TODO(ritch) support other directories
-      if(configFile && configFile.getFacetName() === facetName
-        && configFile.getDirName() === 'models') {
+      if (configFile && configFile.getFacetName() === facetName &&
+        configFile.getDirName() === 'models') {
```

Comment:
https://github.com/strongloop-internal/loopback-knowledge-base/blob/4772f51fcc68c7c1a83157b31eac5379e509d238/style-guide.md#indentation-of-multi-line-expressions-in-if

Indent the second line by two levels.